main.go

```diff
@@ -132,6 +148,23 @@ func main() {
 		}
 	}
 
+	// Log >1GB paths
+	pathsExceedingGB := 0
+	for pth := range pathToIndicatorPath {
+		dirSize, err := dirSize(pth)
+		if err == nil && dirSize > 1073741824 {
+			pathsExceedingGB++
+			log.Debugf("Found a path exceeding 1 GB: %s", pth)
```

Comment:
if a file exceeds 1GB, the path to the file and all parent folders will be included here. Is this the way this should work?